src/ImageRunModal.jsx

```diff
@@ -543,11 +543,11 @@ export class ImageRunModal extends React.Component {
     }
 
     onPlusOne(key) {
-        this.setState(state => ({ [key]: state[key] + 1 }));
+        this.setState(state => ({ [key]: Number(state[key]) + 1 }));
```

Comment:
This PR uses parseInt() and Number() -- what's the main difference here?

Reply:
Switched to using parseInt everywhere